pubnub-kotlin/pubnub-kotlin-api/src/jsMain/kotlin/Pubnub.d.kt

```diff
@@ -348,6 +348,8 @@ open external class PubNub(config: Any /* UUID | UserId */) {
     }
 
     interface `T$6` {
```

Comment:
yes, this whole file is generated from the PN JS SDK Typescript bindings. 
As I go through this file and start using these interfaces, I rename them to something more useful.
However, they will not be present in any generated code (because they're marked `external`) and so these names will not be visible to anyone.